srcs/client/src/Pages/Profile/Fetch/getFetchFriends.tsx

```diff
@@ -0,0 +1,20 @@
+
+export async function getFetchFriends(props : any){
```

Comment:
what's the difference between getFetch and getFetchFriends and getFetchMatch and getFetchSucces... ? I think we should pass the path as a param for example getFetch('/users/'); and have the prefix be constant (for exemple getting it from .env file on start)